ethereumj-core/src/main/java/org/ethereum/sync/PeersPool.java

```diff
@@ -193,7 +194,13 @@ public void connect(Node node) {
         }
 
         synchronized (pendingConnections) {
-            ethereum.connect(node);
```

Comment:
Hi Adrian, 
sounds reasonable. 
Could we move the connect method from the `EthereumImpl` to the `ChannelManager` and use it from both places? It seems to me the most appropriate place?